update.sh

```diff
@@ -45,16 +45,26 @@ generated_warning() {
 	EOH
 }
 
+# Equivalent to PHP's PHP_VERSION_ID:
+# 5.6.10 => 50610, 7.1.0 => 70100
+as_version_id() {
+	local fullVersion="$1"
+	# strip prelease suffix
+	fullVersion="${fullVersion/[a-z]*/}"
```

Comment:
MIght be safer to use `"${fullVersion%[a-z]*}"` so that it only matches the end?